vmdb/app/models/mixins/has_many_orchestration_stack_mixin.rb

```diff
@@ -7,8 +7,8 @@ module HasManyOrchestrationStackMixin
              :dependent   => :destroy
 
     has_many :direct_orchestration_stacks,
+             -> { where(:orchestration_stacks => {:ancestry => nil}) },
```

Comment:
Ah, I interpreted your comment differently from @Fryguy. In that case, I'll go with his response. :wink:

But, if it works, I think we should go with that simpler spelling @kbrock.